src/main/java/kata/makers/model/User.java

```diff
@@ -0,0 +1,15 @@
+package kata.makers.model;
+
+import kata.makers.enums.Role;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@AllArgsConstructor
+@Getter
+@Setter
+public class User {
```

Comment:
Same question as above

Reply:
In FinancialService, I did use setter function for balance
I can put the annotation to the field that I need setter / getter
It should be less confused